import_flow_events.py

```diff
@@ -54,22 +59,63 @@
       utmTerm VARCHAR(40)
     );
 """
+Q_CREATE_METADATA_TABLE = """
+    CREATE TABLE IF NOT EXISTS flow_metadata (
+      flowId VARCHAR(64) NOT NULL UNIQUE SORTKEY,
+      beginTime TIMESTAMP NOT NULL,
+      duration INTERVAL NOT NULL,
+      completed BOOLEAN NOT NULL,
+      newAccount BOOLEAN NOT NULL,
+      browser VARCHAR(40),
+      browserVersion VARCHAR(40),
+      os VARCHAR(40),
+      context VARCHAR(40),
+      entrypoint VARCHAR(40),
+      migration VARCHAR(40),
+      service VARCHAR(40),
+      utmCampaign VARCHAR(40),
+      utmContents VARCHAR(40),
+      utmMedium VARCHAR(40),
+      utmSource VARCHAR(40),
+      utmTerm VARCHAR(40)
+    );
+"""
+Q_CREATE_EVENTS_TABLE = """
+    CREATE TABLE IF NOT EXISTS flow_events (
+      timestamp TIMESTAMP NOT NULL SORTKEY,
+      flowTime INTERVAL NOT NULL,
+      flowId VARCHAR(64) NOT NULL DISTKEY,
+      type VARCHAR(30) NOT NULL
+    );
+"""
 
-Q_CLEAR_EVENTS = """
+Q_CHECK_FOR_DAY = """
+    SELECT timestamp FROM flow_events
+    WHERE timestamp::date >= {day}
```

Comment:
Postgres' casting operator lets us ditch the `day_to_timestamp` function further down, because we can compare against the day string directly.